terraform-dev/cluster.tf

```diff
@@ -51,7 +55,14 @@ resource "libvirt_volume" "root" {
   name = "root-disk-${count.index}.qcow2"
   base_volume_id = element(libvirt_volume.os-qcow2.*.id, count.index)
   pool = "default"
-  size = var.disk_size
+  size = var.root_disk_size
+  count = var.nodes_count
+}
+
+resource "libvirt_volume" "data" {
+  name = "data-disk-${count.index}.qcow2"
+  pool = "default"
```

Comment:
nit: maybe add an option to override the pool with a default to `default`?

Reply:
good point